helm-charts/konk/templates/deployment.yaml

```diff
@@ -67,6 +67,8 @@ spec:
               path: /healthz
               port: https
               scheme: HTTPS
+            failureThreshold: 30
```

Comment:
`30 * 10s = 5m` to finish startup. Seems fine.

Reply:
I skimmed the docs while doing this and found this in an example. I don't think this impacts the liveness probe that was causing me issues